PKPD/gui/simulation.py

```diff
@@ -130,6 +174,105 @@ def _get_data_labels(self):
             return patient_ID_label, time_label, state_labels, dose_label
 
 
+    def get_dose_schedule(self):
+        """Get dose schedule from data, if provided.
+        """
+        # if no dose schedule is provided, set dose schedule to None for each patient
```

Comment:
This function relies on patient IDs starting at 1 and being contiguous. Is there anything that will assure this?

Reply:
Yes, very true indeed. Bug fixed.